staging/src/github.com/clusterpedia-io/api/cluster/v1alpha2/types.go

```diff
@@ -107,6 +107,10 @@ type ClusterGroupResources struct {
 }
 
 type ClusterStatus struct {
+	// +required
+	// +kubebuilder:validation:Required
+	APIServer string `json:"apiserver,omitempty"`
```

Comment:
The required field must not have an omitmepty tag, otherwise the update will fail if it equals ""